src/Refactoring-UI/RBRenameInstanceVariableDriver.class.st

```diff
@@ -0,0 +1,83 @@
+"
+I'm a model for user interaction for the rename method refactoring.
+"
+Class {
+	#name : 'RBRenameInstanceVariableDriver',
+	#superclass : 'RBInteractionDriver',
+	#instVars : [
+		'oldName',
+		'class',
+		'newName',
+		'shouldEscape',
+		'lastName'
+	],
+	#category : 'Refactoring-UI-Drivers',
+	#package : 'Refactoring-UI',
+	#tag : 'Drivers'
+}
+
+{ #category : 'execution' }
+RBRenameInstanceVariableDriver >> applyChanges [ 
+	refactoring privateTransform.
+	refactoring performChanges
```

Comment:
Also, this refactoring doesn't preview the changes. It might be nice to show changes since we're updating references as well, I'm not sure. Old implementation didn't show preview. @Ducasse what do you think?

Take a look at remove class driver:

```suggestion

	self openPreviewWithChanges: refactoring privateTransform
```

Reply:
I didn't implement the change, but just for the record, when I tried the suggested code I got the following error 
<img width="1501" alt="image" src="https://github.com/pharo-project/pharo/assets/4822303/93bf1dd5-710c-4d57-876a-66bbab73b792">

Reply:
Okay, thanks for the report. Probably the changes are missing.